### ISSUE TITLE
Add ability to alter database engine port

### ISSUE THREAD

**Issue by tiukhtinvladimir** [db-migrate/node-db-migrate#567]
## I'm submitting a...
 - [ ] Question

Is it possible to alter database port specifying ```'port': 3308``` like we are doing with ```'host': 'localhost'```?

**Comment by wzrdtales** [comment]
Yes, it depends on the driver you use. But it should be mostly just adding port to the config.
